fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -31,6 +31,12 @@
 
 class FiwareRegionsObjectStorageTests(fiware_region_base_tests.FiwareRegionsBaseTests):
 
+    def setUp(self):
+        """
+        Setup the object storage tests
+        """
+        self.swift_operations.init_swift_client()
+
     def test_create_get_and_delete_container(self):
```

Comment:
Why initializing the Swift client before *every* test. Please turn this into a ``setUpClass`` classmethod, and don't forget to invoke ``super``. An example can be found at ``fiware_region_base_tests``